points_tracker/views.py

```diff
@@ -111,24 +112,24 @@ def upload_file():
                     pass
                 file.save(filepath)
 
+                #load the file
                 if extension == ".mp3":
-                    length = int(round(MP3(filepath).info.length))
-                    sound = AudioSegment.from_mp3(filepath)
-                    sound.export(filepath, format="wav")
-                elif extension == ".mp4":
-                    length = int(round(MP4(filepath).info.length))
-                    sound = AudioSegment.from_file(filepath)
-                    sound.export(filepath, format="wav")
+                    audiofile = AudioSegment.from_mp3(filepath)
                 elif extension == ".wav":
-                    length = 0
+                    audiofile = AudioSegment.from_wav(filepath)
                 else:
-                    length = 0
+                    audiofile = AudioSegment.from_file(filepath)
+
+                audiofile = audiofile + -(audiofile.dBFS - current_app.config['AUDIO_DB_LEVEL'])    #level the file
+                audiofile = audiofile[:current_app.config['MAX_AUDIO_DURATION']]                    #trim the file
```

Comment:
this should fix our issue with people uploading long files it just keeps the first n seconds of the file if its too long